tekton/src/pipelines/gitops/deprovision-mas-instance.yml.j2

```diff
@@ -186,3 +192,91 @@ spec:
       taskRef:
         kind: Task
         name: gitops-process-mongo-user
+{% if deprovision_manage_addons == true %}
+    - name: gitops-deprovision-efs
+      runAfter:
+        - gitops-process-mongo-user
+      params:
+        - name: cluster_name
+          value: $(params.cluster_name)
+        - name: avp_aws_secret_region
+          value: $(params.avp_aws_secret_region)
+        - name: avp_aws_secret_key
+          value: $(params.avp_aws_secret_key)
+        - name: avp_aws_access_key
+          value: $(params.avp_aws_access_key)
+        - name: mas_instance_id
+          value: $(params.mas_instance_id)
+        - name: cloud_provider
+          value: $(params.cloud_provider)
+      workspaces:
+        - name: configs
+          workspace: configs
+      taskRef:
+        kind: Task
+        name: gitops-deprovision-efs
+    - name: gitops-delete-kafka-config
+      runAfter:
+        - gitops-deprovision-efs
+      params:
+        - name: cluster_name
+          value: $(params.cluster_name)
+        - name: account
+          value: $(params.account)
+        - name: secrets_path
+          value: $(params.secrets_path)
+        - name: mas_instance_id
+          value: $(params.mas_instance_id)
+        - name: git_branch
+          value: $(params.git_branch)
+        - name: github_org
+          value: $(params.github_org)
+        - name: github_repo
+          value: $(params.github_repo)
+        - name: github_host
+          value: $(params.github_host)
+        - name: github_pat
+          value: $(params.github_pat)
+        - name: avp_aws_secret_region
+          value: $(params.avp_aws_secret_region)
+        - name: avp_aws_secret_key
+          value: $(params.avp_aws_secret_key)
+        - name: avp_aws_access_key
+          value: $(params.avp_aws_access_key)
+      workspaces:
+        - name: configs
+          workspace: configs
+      taskRef:
+        kind: Task
+        name: gitops-delete-kafka-config
+    - name: gitops-deprovision-kafka
```

Comment:
There is one kafka per cluster so gitops-deprovision-kafka needs to be in the deprovision-cluster pipeline

Reply:
moved the kafka uninstall to deprovision-cluster.yml.j2